source/guides/models/repositories.md

```diff
@@ -95,6 +95,7 @@ class BookRepository < Hanami::Repository
       .where(author_id: author.id)
       .order(:published_at)
       .limit(limit)
+      .to_a
   end
 end
```

Comment:
1. yep, I want to see equal style in this places (with or without `#to_a`)
2. I think we should add simple description about `#to_a` method before all this